src/main/resources/db/migration/V49.0__DML_update_config_table.sql

```diff
@@ -0,0 +1,393 @@
+  },
+  {
+    "name": "Settings",
+    "icon": "settings",
+    "key": 6,
+    "submenu": [
+      {
+        "name": "Companies",
+        "href": "/#/companies",
+        "key": 61
+      },
+      {
+        "name": "Countries",
+        "href": "/#/countries",
+        "key": 62
+      },
+      {
+        "name": "Holidays",
+        "href": "/#/holidays",
+        "key": 63
+      },
+      {
+        "name": "Leave Types",
+        "href": "/#/leave-types",
+        "key": 64
+      },
+      {
+        "name": "Projects",
+        "href": "/#/projects",
+        "key": 65
+      },
+      {
+        "name": "Project types",
+        "href": "/#/project-types",
+        "key": 66
+      },
+      {
+        "name": "Roles",
+        "href": "/#/roles",
+        "key": 67
+      },
+      {
+        "name": "Seniorities",
+        "href": "/#/seniorities",
+        "key": 68
+      },
+      {
+        "name": "Technologies",
+        "href": "/#/technologies",
+        "key": 69
+      },
+      {
+        "name": "Tenants",
+        "href": "/#/tenants",
+        "key": 611
+      }
+    ]
+  },
+  {
+    "name": "Reports",
+    "icon": "reports",
+    "key": 7,
+    "submenu": [
+      {
+      	"name": "Ptos by Employee",
+        "href": "/#/reports/ptos/employees",
+        "key": 72
+      },
+            {
+      	"name": "Ptos by Client",
+        "href": "/#/reports/ptos/clients",
+        "key": 73
+      }
+    ]
+  }
+]'::json,permissions='[
+  {"entity": "employees", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "contracts", "actions": ["create", "read", "delete"]},
+  {"entity": "assignments", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "clients", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "companies", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "projects", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "project-types", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "roles", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "seniorities", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "technologies", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "tenants", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "leave-types", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "holidays", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "countries", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "vacations", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "ptos", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "reports/employees", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "reports/ptos/employees", "actions": ["create", "read", "update", "delete"]}
+]'::json
+WHERE id='3c245af9-19fc-4e02-b9b0-a85e8ed06594'::uuid::uuid;
```

Comment:
Minor: it's useful for review to add a comment indicating which role we're updating. Not for this pr, but for next